src/plugins/report_RHTSupport_AddData.xml.in

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<event>
+    <_name>Red Hat Customer Support</_name>
+    <_description>Attach the data to existing Red Hat support case</_description>
+
+    <requires-items>package</requires-items>
+    <requires-details>yes</requires-details>
+    <exclude-items-by-default>event_log,vmcore</exclude-items-by-default>
```

Comment:
Maybe, you should not exclude vmcore because this the type of file support engineers asks for often.

Reply:
good idea, updated